src/main/java/net/masterthought/cucumber/json/Result.java

```diff
@@ -31,4 +31,9 @@ public String getFormatedDuration() {
     public String getErrorMessage() {
         return StringEscapeUtils.escapeHtml(errorMessage);
     }
+
+    // method used in Velocity templating library
+    public String getErrorMessageTitle() {
+        return errorMessage.split("\\r\\n|\\r|\\n")[0];
```

Comment:
Now method is final.

If first line very long it will be splitted to multiple lines